service-workers/service-worker/active.https.html

```diff
@@ -5,51 +5,55 @@
 <script src="resources/test-helpers.sub.js"></script>
 <body>
 <script>
+
+const SCRIPT = 'resources/empty-worker.js';
+const SCOPE = 'resources/blank.html';
+let registration;
+
 // "active" is set
-async_test(function(t) {
-    var step = t.step_func.bind(t);
-    var url = 'resources/empty-worker.js';
-    var scope = 'resources/blank.html';
-    var frame;
-    var registration;
+promise_test(async t => {
+    let step = t.step_func.bind(t);
+    let frame;
+
+    await service_worker_unregister(t, SCOPE);
+    frame = await with_iframe(SCOPE);
+    registration =
+        await navigator.serviceWorker.register(SCRIPT, {scope: SCOPE});
+    await wait_for_state(t, registration.installing, 'activating');
+    const container = await frame.contentWindow.navigator.serviceWorker;
+    assert_equals(
+        container.controller,
+        null,
+        'On activating state a document should not have a controller');
+    assert_equals(
+        registration.active.scriptURL,
+        normalizeURL(SCRIPT),
+        'On activating state a document should have an active worker ');
```

Comment:
Similarly, documents don't have acive/waiting/installing workers (that was an old version of the spec). I should just make the desription 'registration.active'. Ditto below.